src/Extractor/BaseExtractor.php

```diff
@@ -172,21 +169,16 @@ protected function isAlive(): void
     {
         try {
             $this->testConnection();
-        } catch (\Throwable $e) {
+        } catch (Throwable $e) {
             throw new DeadConnectionException('Dead connection: ' . $e->getMessage());
         }
     }
 
-    protected function handleDbError(Throwable $e, ?array $table = null, ?int $counter = null): UserException
+    protected function handleDbError(Throwable $e, ExportConfig $export): UserException
     {
-        $message = '';
-        if ($table) {
```

Comment:
Pouziva sa to iba na jednom mieste a `$table` (config) a `outputTable` bola vzdy definovana - aj v povodnom kode, ... preto je tato podmienka odstranená.